// graphql-shield/cjs/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.not = exports.or = exports.race = exports.chain = exports.and = exports.deny = exports.allow = exports.inputRule = exports.rule = exports.shield = void 0;
var shield_js_1 = require("./shield.js");
Object.defineProperty(exports, "shield", { enumerable: true, get: function () { return shield_js_1.shield; } });
var constructors_js_1 = require("./constructors.js");
Object.defineProperty(exports, "rule", { enumerable: true, get: function () { return constructors_js_1.rule; } });
Object.defineProperty(exports, "inputRule", { enumerable: true, get: function () { return constructors_js_1.inputRule; } });
Object.defineProperty(exports, "allow", { enumerable: true, get: function () { return constructors_js_1.allow; } });
Object.defineProperty(exports, "deny", { enumerable: true, get: function () { return constructors_js_1.deny; } });
Object.defineProperty(exports, "and", { enumerable: true, get: function () { return constructors_js_1.and; } });
Object.defineProperty(exports, "chain", { enumerable: true, get: function () { return constructors_js_1.chain; } });
Object.defineProperty(exports, "race", { enumerable: true, get: function () { return constructors_js_1.race; } });
Object.defineProperty(exports, "or", { enumerable: true, get: function () { return constructors_js_1.or; } });
Object.defineProperty(exports, "not", { enumerable: true, get: function () { return constructors_js_1.not; } });
